meta-refkit-core/lib/oeqa/selftest/cases/refkit_ostree.py

```diff
@@ -99,7 +99,7 @@ def create_httpd():
 #exec socat 2>>/tmp/ostree.log -D -v -d -d -d -d STDIO TCP:localhost:%d
 ''' % port)
 
-            cmd = '''ostree config set 'remote "updates".url' http://%s && refkit-ostree update''' % self.OSTREE_SERVER
+            cmd = '''ostree config set 'remote "updates".url' http://%s && refkit-ostree-update -O -P "/usr/share/refkit-ostree/hooks/post-apply.d/00-update-uefi-app"''' % self.OSTREE_SERVER
```

Comment:
Can you use the long version of the command line options (--one-shot and --post-apply-hook) here and explain in the source code (instead of the commit message) why it is done this way?

The comment about "only run UEFI app update hook in update selftest" will be easier to understand when it also mentions what is *not* run, i.e. the automatic reboot hook.